tests/communication_1to1_test.py

```diff
@@ -36,6 +38,8 @@ def setup_streams(input_streams):
 
     @frequency(1)
     def publish_msg(self):
+        if self._cnt == MAXIMUM_COUNT:
+            sys.exit()
```

Comment:
Does this work in Ray? I think it crashes the actor. IMHO, we need a different solution here. We need a mechanism to exit an actor. On the publisher side we could do:

```
def execute(self):
  for i in range(0, MAX_MSG_COUNT:
    publish_msg()
    time.sleep(1)
```
We'll have to change `ros_executer` to ensure that we do not spin.